packages/hint-scoped-svg-styles/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "description": "Scoped SVG Styles checks if SVG styles affect any other elements outside the svg.",
+  "dependencies": {
+    "@hint/utils": "^3.1.1",
+    "postcss": "^7.0.17"
```

Comment:
I think that if `postcss` is used only for types, it can be a devDependency. Am I right @antross?

Reply:
Right. It should be in `devDependencies`. I am using only `Rule` type from it.
Changes done.